nmt.go

```diff
@@ -265,11 +265,11 @@ func (n *NamespacedMerkleTree) Push(namespacedData namespace.PrefixedData) error
 
 // Return the namespaced Merkle Tree's root together with the
 // min. and max. namespace ID.
-func (n *NamespacedMerkleTree) Root() namespace.IntervalDigest {
+func (n *NamespacedMerkleTree) Root() []byte {
```

Comment:
Maybe the godoc comment should now be more explicit that the returned bytes are min, max and the hash concatenated, i.e. `min||max||hashDigest`?

Reply:
good catch! resolved in https://github.com/celestiaorg/nmt/pull/48/commits/464e30105194c9653c2f7f0c6569a03bc95080f3